practioners/README.md

```diff
@@ -0,0 +1,198 @@
+| name\[0].prefix\[0]                               | `Agent / AgentPerson / name / prefix`                                                         |
+
+<details>
+    <summary>Example JSON</summary>
+
+```
+{
+    "resource": {
+        "resourceType": "Practitioner",
+        "id": "C5DEFBF3-0174-BC6F-182C-B777B9C6FF43",
+        "meta": {
+            "profile": [
+                "https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-Practitioner-1"
+            ]
+        },
+        "name": [
+            {
+                "use": "official",
+                "family": "Doe",
+                "given": [
+                    "John"
+                ],
+                "prefix": [
+                    "Dr"
+                ]
+            }
+        ]
+    }
+}
+```
+
+</details>
+
+### PractitionerRole
+
+A `PractionerRole` resource will only be added if the HL7 `AgentPerson` has an associated `representedOrganization`.
+
+| Mapped to (JSON FHIR PractitionerRole resource field) | Mapped from (XML HL7 / other)                                                                             |
+|-------------------------------------------------------|-----------------------------------------------------------------------------------------------------------|
+| id                                                    | `Agent / id [@root]` appended with `-PR`                                                                  |
+| meta                                                  | fixed value = `"https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-PractitionerRole-1"`         |
+| practitioner.reference                                | Reference to the mapped Practitioner (see above)                                                          |
+| organization.reference                                | Reference to the [Organisation](../organisations/README.md) mapped from `Agent / representedOrganization` |
+| code\[0].text                                         | `Agent / code / orginalText` or else `Agent / code [@displayName]`                                        |
+
+<details>
+    <summary>Example JSON</summary>
+
+```
+{
+    "resource": {
+        "resourceType": "PractitionerRole",
+        "id": "94F00D99-0601-4A8E-AD1D-1B564307B0A6-PR",
+        "meta": {
+            "profile": [
+                "https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-PractitionerRole-1"
+            ]
+        },
+        "practitioner": {
+            "reference": "Practitioner/94F00D99-0601-4A8E-AD1D-1B564307B0A6"
+        },
+        "organization": {
+            "reference": "Organization/94F00D99-0601-4A8E-AD1D-1B564307B0A6-ORG"
+        },
+        "code": [
+            {
+                "text": "General Medical Practitioner"
+            }
+        ]
+    }
+}
+```
+
+</details>
+
+### Unmapped fields
```

Comment:
Minor feedback/opinion: I feel having the unmapped fields just below each of the two resource types would be more cohesive as opposed to grouping them together here.